code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Requirements.hs

```diff
@@ -131,8 +132,4 @@ understandability = cic "understandability" (foldlSent [
   ]) "Understandability" nonFuncReqDom
 
 maintainability :: ConceptInstance
-maintainability = cic "maintainability" (foldlSent [
-  S "development time for any" `S.the_ofTheC` S "likely changes should not exceed", 
-  addPercent (10 :: Integer), S "percent of the original development time"
-  ]) "Maintainability" nonFuncReqDom
-
+maintainability = mkMaintainableNFR "maintainability" 10 "Maintainability"
```

Comment:
Sorry to be the nitpick... can this standard maintainability be moved to `drasil-docLang` now too?

Reply:
I'm not sure what you mean. The code you linked is a call to the function exported from `drasil-docLang`. How would I go about moving it to `drasil-docLang`? Additionally, the parameters are not consistent across all the examples.

Reply:
Discussed in person. We'll just move this to a new ticket.